lib/fs.js

```diff
@@ -1460,17 +1460,19 @@ fs.unwatchFile = function(filename, listener) {
 
 // Regexp that finds the next partion of a (partial) path
 // result is [base_with_slash, base], e.g. ['somedir/', 'somedir']
+var nextPartRe;
 if (isWindows) {
-  var nextPartRe = /(.*?)(?:[\/\\]+|$)/g;
+  nextPartRe = /(.*?)(?:[\/\\]+|$)/g;
 } else {
-  var nextPartRe = /(.*?)(?:[\/]+|$)/g;
+  nextPartRe = /(.*?)(?:[\/]+|$)/g;
 }
 
```

Comment:
I thought about doing that, but wanted to keep the change as small as reasonable. But that was a guideline I applied and not a hard rule, so yeah, I'm happy to make that change.